apps/nouns-camp/src/components/action-dialog.js

```diff
@@ -247,6 +250,25 @@ const createSignature = (functionAbiItem) => {
   return `${functionAbiItem.name}(${formattedInputs.join(",")})`;
 };
 
+const useNounIdsByOwner = ({ owner } = {}) => {
```

Comment:
Two things about this:

1. Not sure where to put it 😅 token-contract or store seem wrong
2. I could just return nouns (not only ids) if we were to use the hook somewhere else, but felt unnecessary right now. can easily be changed afterwards